client/components/SpeechToText/SpeechToText.js

```diff
@@ -32,6 +32,10 @@ export default function SpeechToText({ game, handleMove, handleEat, curPos }) {
     resetTranscript();
   };
 
+  const handleClearTranscript = () => {
+    resetTranscript();
+  };
+
   const commands = [
     {
       command: 'move :word',
```

Comment:
Having seen you demo it, it is awesome to know you could also give more commands in the future if you wanted to!